wake/loaders/productListingPage.ts

```diff
@@ -214,7 +214,8 @@ const searchLoader = async (
 
   const hasPreviousPage = page > 1;
 
-  const pageOffset = props.pageOffset ?? 1;
+  // if the home page is not 0, there is a chance that the "previous page" button will replicate the value of the current page
+  const pageOffset = 0;
```

Comment:
```suggestion
  const pageOffset = props.pageOffset ?? 0;
```
You can set the `pageOffset` prop to be zero. It is ok to set the default to zero for this loader but you still need to use this property if needed.

Also, I believe that hasPreviousPage should be: `hasPreviousPage = page > pageOffset`.